### PR TITLE
Updated the instructions to use the new Parter Integration Workflow

### PR DESCRIPTION
**Pull request by caiusb**
Update to use the new partner integration workflow.

Replaced S3 input with SFTP and removed the S3+Glue section.

### CODE REVIEW

**Comment by caiusb** [comment]
This will need another pass as I only wrote it, and I didn't read it end-to-end yet

**Review by romming — CHANGES_REQUESTED** [review]

File: README.md [hunk 1]
```diff
@@ -12,94 +12,68 @@ In this workshop you'll learn how to create a Redshift data warehouse that centr
...
+Click "Create cluster".
+It will take 5-10 minutes for the cluster to start up.
 
-## 2. Connect Etleap to Redshift and the data sources
+## 2. Connect Etleap to Redshift and the data sources, and set up pipelines
 
-We'll ETL data from two different data sources into your Redshift data warehouse. One source, an S3 bucket, contains JSON-formatted click event data. The other, a MySQL database, contains information about users. 
+We'll ETL data from two different data sources into your Redshift data warehouse. One source, an SFTP source, contains JSON-formatted click event data. The other, a MySQL database, contains information about users. 
 
 The first thing you'll need to do is log into Etleap using the credentials that were provided to you via email. The email has the subject line 'Your Etleap Account'.
```
Comment (romming) [inline README.md:35]: No longer correct, right?

File: README.md [hunk 2]
```diff
@@ -12,94 +12,68 @@ In this workshop you'll learn how to create a Redshift data warehouse that centr
...
 
 In the rest of this section we'll connect Etleap to the data sources and Redshift destination, so that we can create pipelines in the next section.
 
+### 2.1. Set up the Redshift connection via the Partner Integration Console
```
Comment (romming) [inline README.md:39]: Would it make sense for this to be section 2 and the pipeline setup be section 3?

**Review by romming — CHANGES_REQUESTED** [review]
I can review again after the mongo changes.

File: README.md (hunk 2 above)
Comment (caiusb) [reply README.md]: I have made each source be its own section.

File: README.md (hunk 1 above)
Comment (caiusb) [reply README.md]: Correct. Removed.

File: README.md [hunk 3]
```diff
@@ -9,140 +9,150 @@ In this workshop you'll learn how to create a Redshift data warehouse that centr
...
+- Go to the Redshift Console [here](https://console.aws.amazon.com/redshiftv2/home?region=us-east-1#clusters).
+- Select the cluster your just created. 
+- In the top right corner, click on the "Add Partner Integration" button.
+- From the list of partners, select "Etleap". It should be the first option. Click "Next".
+- Leave all the settings as they are and click "Add partner".
+- This will take you to the Etleap console.
+  - Confirm your Redshift password. Use the same password you used when you created the Redshift Cluster. Click "Validate and Setup Connection."
+  - Add your email address. This will send you a confirmation email. Click the link in the email to continue.
+- Fill in your details, and click "Create Account!"
+- Congratulation! Your account and connection are now ready to use. We'll go ahead, and set up more connections.
```
Comment (romming) [inline README.md:60]:
```suggestion
- Congratulations! Your Etleap account and Redshift connection are now ready to use.
```

File: README.md [hunk 4]
```diff
@@ -9,140 +9,150 @@ In this workshop you'll learn how to create a Redshift data warehouse that centr
...
-- Additional properties: Leave as their defaults.
+- Name: `Website Events`
+- Hosts and Ports: `test.dev.etleap.com:27017`
+- Database: `webstore`
+- Authentication Database: `test` 
+- Username: `devdays`
+- Password: `i5eU2nZx4d`
 
-Click 'Create Connection'.
+Let the 2 checkboxes unchecked.
```
Comment (romming) [inline README.md:78]:
```suggestion
Leave the 2 checkboxes unchecked.
```

File: README.md [hunk 5]
```diff
@@ -9,140 +9,150 @@ In this workshop you'll learn how to create a Redshift data warehouse that centr
...
 
-## 3. Create Etleap ETL pipelines from the sources to Redshift
+Click 'Save'. 
 
-In this section we'll configure pipelines that will ETL data from the sources into the Redshift database.
+This will take you to the list of available collection in the MongoDB database.
```
Comment (romming) [inline README.md:82]:
```suggestion
This will take you to the list of available collections in the MongoDB database.
```

**Review by romming — APPROVED** [review]
Looks good!

**Pull request merged**